cli/src/alluxio.org/cli/env/env_vars.go

```diff
@@ -29,6 +30,7 @@ type AlluxioConfigEnvVar struct {
 	description string // describes the environment variable in conf/alluxio-env.sh.template file
 
 	// optional
+	IsJavaOpts                bool              // true if environment variable's value is a java opts string
```

Comment:
i didn't find any usage of this bool. why is this necessary?

Reply:
agh i forgot to use it x.x